jobs/fixtures/response.json

```diff
@@ -0,0 +1,9 @@
+{
+  "status": "succeed",
+  "jobs": [
+    {},
+    {},
+    {}
+  ],
+  "": 3
+}
```

Comment:
it would be nice to have a snapshot of the response.

Reply:
added